src/actions/index.js

```diff
@@ -12,4 +12,11 @@ const RemoveBook = book => (
   }
 );
 
-export { CreateBook, RemoveBook };
+const ChangeFilter = category => (
+  {
+    type: 'CHANGE_FILTER ',
```

Comment:
**OPTIONAL**: Why is there a space? Nothing wrong with that but very unusual.

You could defined the action types as constants here and export them to use them also on the reducers, that way you can avoid typos.

```js
const CHANGE_FILTER = 'CHANGE_FILTER';
const CREATE_BOOK = 'CREATE_BOOK';
// ...
// ...
// ...
const changeFilter = category => ({
  type: CHANGE_FILTER,
  category,
});

export { CHANGE_FILTER, CREATE_BOOK, ... }
```